src/utils/generate-address.ts

```diff
@@ -26,7 +26,7 @@ export interface GenerateAddressOptions {
   includeClassicAddress?: boolean
 }
 
-function generateAddressAPI(options: GenerateAddressOptions = {}): GeneratedAddress {
+function generateXAddress(options: GenerateAddressOptions = {}): GeneratedAddress {
```

Comment:
This should actually be moved to the `Wallet` class as a static method. But that's probably out of the scope of this PR, so you can just add a `TODO` for now.

Reply:
Added todo